lesscss/improved-initiative.less

```diff
@@ -127,6 +127,10 @@ h4 {
   font-size: @font-size-h4;
 }
 
+ul {
```

Comment:
Can you move this rule to an appropriate place in `statblock.less` to ensure that it is more conservatively scoped? Thanks!

Reply:
Ok it's targeted at ol / ul inside that .power-content class which has to be inside a .c-statblock class